style.css

```diff
@@ -0,0 +1,85 @@
+.ul {
+  list-style: none;
+}
+
+.main-section {
+  background-image: url('img/bg.png');
+  background-size: cover;
+  background-repeat: no-repeat;
+  min-height: 800px;
+}
+
+.text-section {
+  margin: 0 260px 0 260px;
+  padding-top: 200px;
```

Comment:
- Let's reduce this padding-top also, it's making it have too much space. let's adjust our design here also.